design/app-to-app.md

```diff
@@ -0,0 +1,87 @@
+App to App Interactions
+===========
+
+This document describes an extension to the SDK to allow MRE apps to receive content (e.g., photos) from the MRE clients. Content may be generated by the client itself, or, in future iterations not yet described here, may arrive from other apps with coordination from a particular client.
```

Comment:
Please describe why this feature is useful, expected, or otherwise adds to the MRE SDK's value.

Reply:
good call. I'll explain the photo scenario.